package.json

```diff
@@ -23,6 +25,7 @@
     "eslint-config-prettier": "^8.7.0",
     "eslint-plugin-react": "^7.32.2",
```

Comment:
Add rules for your linter or remove the script if your feature doesn't use this.

Reply:
Done.